Dockerfile

```diff
@@ -104,16 +104,16 @@ COPY . .
 COPY --from=deps /workspace-install ./
 
 # Optional: if the app depends on global /static shared assets like images, locales...
-RUN yarn workspace $PROJECT share-static-hardlink && yarn workspace $NEXT_PUBLIC_CMS_GRAPHQL_ENDPOINT build
```

Comment:
Looks like I left accidentally some debug stuff there. Good that you fixed them back.